client/components/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Overview from './overview/Overview.jsx';
+import ReviewsList from './reviews/ReviewsList.jsx';
+
+const App = () => (
+  <div>
+    <div><Overview/></div>
```

Comment:
Is there a reason you are wrapping your React components inside a div element? These div wrappers may be unnecessary.

Reply:
I was getting an error message without wrapping that said JSX elements can't be rendered adjacent to each other and need to be wrapped in a containing element, such as a DIV.

Reply:
interesting, gotcha!